File: app/src/main/java/com/martinlaizg/geofind/utils/KeyboardUtils.java
package com.martinlaizg.geofind.utils;

import android.app.Activity;
import android.content.Context;
import android.view.View;
import android.view.inputmethod.InputMethodManager;

public class KeyboardUtils {

	/**
	 * Hide the keyboard
	 *
	 * @param activity
	 * 		the activity
	 */
	public static void hideKeyboard(Activity activity) {
		InputMethodManager editTextInput = (InputMethodManager) activity
				.getSystemService(Context.INPUT_METHOD_SERVICE);
		View currentFocus = activity.getCurrentFocus();
		if(currentFocus != null && editTextInput != null) {
			editTextInput.hideSoftInputFromWindow(currentFocus.getWindowToken(), 0);
		}
	}
}

File: app/src/main/java/com/martinlaizg/geofind/utils/KeyboardUtils.kt
package com.martinlaizg.geofind.utils

import android.app.Activity
import android.content.Context
import android.view.inputmethod.InputMethodManager

object KeyboardUtils {
    /**
     * Hide the keyboard
     *
     * @param activity
     * the activity
     */
    fun hideKeyboard(activity: Activity) {
        val editTextInput = activity
                .getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
        val currentFocus = activity.currentFocus
        if (currentFocus != null && editTextInput != null) {
            editTextInput.hideSoftInputFromWindow(currentFocus.windowToken, 0)
        }
    }
}